src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -35,7 +35,7 @@ export const ArtworkApp: React.SFC<Props> = props => {
       <Row>
         <Col sm={8}>
           <Box px={[0, 4]}>
-            <ImageBrowser artwork={props.artwork} />
+            <ArtworkBrowser artwork={props.artwork} />
```

Comment:
How come we renamed that to ArtworkBrowser? Is't that a component that just does the slideshow and zoom on Images. ArtworkBrowser suggests that one will be flipping through artworks in my mind.